src/server.rs

```diff
@@ -437,21 +439,34 @@ fn delete_user(
     Ok(Json(json!({})))
 }
 
-/// ## GET status/heartbeat
-///
-/// Process a `GET /status` request returning a heartbeat `200 Ok` response if all
-/// is well
-#[get("/status")]
-fn status(config: ServerConfig) -> HandlerResult<Json> {
-    let config = config;
-    // TODO: check the database.
+/// ## GET Dockerflow checks
+#[get("/__version__")]
+fn version() -> content::Json<&'static str> {
+    content::Json(include_str!("../version.json"))
+}
 
-    Ok(Json(json!({
-        "status": "Ok",
-        "fxa_auth": config.fxa_host.clone(),
-    })))
+#[get("/__heartbeat__")]
+fn heartbeat(conn: Conn, config: ServerConfig) -> status::Custom<Json> {
+    let (status, db_msg) = match db::health_check(&*conn) {
+        Ok(_) => (Status::Ok, "ok".to_owned()),
+        Err(e) => (Status::ServiceUnavailable, e.description().to_owned()),
```

Comment:
I had a TODO here in megaphone to "sanitize the db error message" but I can't imagine "SELECT 1" producing anything that scary..